master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -808,7 +808,8 @@ private[celeborn] class Master(
         disks,
         userResourceConsumption,
         requestId)
-      logInfo(s"Registered worker $workerToRegister.")
+      logInfo(
+        s"Registered worker: ${workerToRegister.workerInfoToString(conf.humanFriendlyLogEnabled)}.")
```

Comment:
Better to cache the value of `conf.humanFriendlyLogEnabled`. Although logging here is not frequent, I think it's a better practice :)

Reply:
ACK, updated the diff.